src/adapters/scene.js

```diff
@@ -123,6 +123,10 @@ Scene.prototype.initMapBox = function () {
       store.setLastLocation({ lng, lat, zoom })
     })
 
+    this.mb.loadImage(`${baseUrl}statics/images/direction_icons/walking_bullets.png`, (error, image) => {
```

Comment:
* this function `loadImage` breaks the tests as it is not present in "mapbox-gl-js-mock" that we rely on to mock the map. Could you add a mocked implementation in https://github.com/QwantResearch/mapbox-gl-js-mock/blob/master/classes/map.js ?

* Also note that there is a typo in the file name: `walking_bullets` :arrow_right: `walking_bullet`. That causes an error that should be checked before calling `addImage` ;)

Reply:
Here is the PR to make our tests pass https://github.com/QwantResearch/mapbox-gl-js-mock/pull/6 . After the merge, we also need to point to the lastest commit for the dependency `mapbox-gl-js-mock` if i'm not mistaking 👍